gapic/templates/tests/unit/gapic/%name_%version/%sub/test_%service.py.j2

```diff
@@ -1041,9 +1046,21 @@ def test_transport_adc(transport_class):
         transport_class()
         adc.assert_called_once()
 
-{{ test_macros.transport_kind_test(service, opts) }}
+{% set configs = [] %}
+{% for transport_name in opts.transport %}
+    {% do configs.append({'service':service, 'transport_name':transport_name}) %}
+{# TODO: Remove the following guard once support for async REST is GA.
+ # See related issue: https://github.com/googleapis/gapic-generator-python/issues/2121.
+#}
+{% if 'grpc' in transport_name or rest_async_io_enabled %}
+    {% do configs.append({'service':service, 'transport_name':transport_name + "_asyncio"}) %}
+{% endif %}
+{% endfor %}
```

Comment:
nit, to emphasize all these go together: delete the blank line here

Reply:
addressed.